docs/sources/userguide/dockerlinks.md

```diff
@@ -206,12 +206,30 @@ from the URL (e.g. `WEBDB_PORT_8080_TCP_PROTO=tcp`).
 If there are multiple ports exposed then the above set of environment
 variables will be defined for each one.
 
-Finally, there will be an environment variable called `<alias>_PORT` that will
+Additionally, there will be an environment variable called `<alias>_PORT` 
+that will
 contain the URL of the first exposed port of the source container.
 For example, `WEBDB_PORT=tcp://172.17.0.82:8080`. In this case, 'first'
 is defined as the lowest numbered port that is exposed. If that port is
 used for both tcp and udp, then the tcp one will be specified.
 
+Finally, Docker will expose each "Docker managed" environment variable
+from the source container as a new environment variable in the target
+container. 
+Docker managed environment variables are the ones defined by any `ENV`
+command in the Dockerfile used to create the source container, or any defined
+using the `-e` option on the `docker run` command when the source container
+was started.
```

Comment:
If you change it above, this could become: *Environment variables originating from Docker include both any `ENV` commands in the Dockerfile used to create the source source container and those defined with the `-e` option on the `docker run` command..*